src/wix/test/WixToolsetTest.CoreIntegration/MsiFixture.cs

```diff
@@ -873,7 +873,7 @@ public void FailsBuildAtLinkTimeForMissingEnsureTable()
                     first =>
                     {
                         Assert.Equal(MessageLevel.Error, first.Level);
-                        Assert.Equal("The identifier 'WixCustomTable:TableDefinitionNotExposedByExtension' could not be found. Ensure you have typed the reference correctly and that all the necessary inputs are provided to the linker.", first.ToString());
+                        Assert.Equal("Cannot find the table definitions for the 'TableDefinitionNotExposedByExtension' table.  This is likely due to a typing error or missing extension.  Please ensure all the necessary extensions are supplied on the command line with the -ext parameter.", first.ToString());
```

Comment:
Please change the name of the test to match the new expected behavior. `FailsBuildAtBindTimeForMissingEnsureTable`?